components/prize_pool/commons/prize_pool_placement.lua

```diff
@@ -239,7 +239,7 @@ function Placement:_getLpdbData(...)
 			participantlink = Opponent.toName(opponent.opponentData),
 			participantflag = opponentType == Opponent.solo and players.p1flag or nil,
 			participanttemplate = opponent.opponentData.template,
-			opponentindex = opponentIndex, -- Needed in SMW
+			opponentindex = opponentIndex,
```

Comment:
Can this be entirely remove now?

Reply:
i think so but not sure yet, needs checking
it is one of the reasons this is still a draft^^

Reply:
```suggestion
```